spec.yaml

```diff
@@ -43,6 +43,8 @@ tags:
     externalDocs:
       url: https://docs.giantswarm.io/guides/accessing-services-from-the-outside/
       description: "User guide: Accessing Pods and Services from the Outside"
+  - name: managed apps
+    description: "An app is a helm chart that you can easily install on your tenant clusters. Use these endpoints to: find out what catalogs your installation supports, and list and install apps on on your tenant clusters."
```

Comment:
In terms of the topic/tag here, I would suggest to call it `apps` instead of `managed apps`.

Terminology in routes like `/v4/clusters/{cluster_id}/apps/` suggests that we call those things "apps". The fact that they are sometimes managed should not be of importance for the general grouping in the API spec.

Reply:
That makes sense.

Right now we're grouping also the `/v4/appcatalogs/` call into this `managed apps` tag.

The idea was to brand these set of endpoints as a feature, the `managed apps` feature.

That said, in the api docs, it might be fine if we introduce two topics. `apps` and `appcatalog`.

What do you think?

Reply:
If app catalogs exist purely to allow installing apps, then I think it would be fine to group it all under the topic of `apps`. But no strong opinion on this one now.

Let's try whatever you find useful and see if it makes sense in ReDoc.

Reply:
Alright, I'll try grouping it all under `apps`. Will hold of on that change until the installing apps PR is merged into this one. Then I make the change all in one go here.